mkt/langpacks/models.py

```diff
@@ -66,6 +78,44 @@ def path_prefix(self):
     def file_path(self):
         return os.path.join(self.path_prefix, nfd_str(self.filename))
 
+    @property
+    def download_url(self):
+        url = ('%s/langpack.zip' %
+            reverse('downloads.langpack', args=[unicode(self.pk)]))
+        return absolutify(url)
+
+    @property
+    def manifest_url(self):
+        """Return URL to the minifest for the langpack"""
+        if self.active:
+            return absolutify(
+                reverse('langpack.manifest', args=[unicode(UUID(self.pk))]))
```

Comment:
`pk` is whatever field has the `primary_key=True` set (django creates one by default if there isn't one). 

As mentioned in the models file, I needed an `UUIDField` to be able to determine the filename before saving the instance. I thought it was redundant to have another field as the primary key when I already had something unique, so I set `uuid` as the primary key.

The reason it's passed through `UUID()` here is to have manifest urls that look like the ones we use for apps. (`/12345678-1234-5678-1234-567812345678/` vs `/12345678123456781234567812345678/`).